### PR TITLE
Qrda 5 2 errata

### PR DESCRIPTION
**Pull request by dczulada**
Pull requests into cqm-parsers require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [ ] This pull request describes why these changes were made.
- [ ] Internal ticket for this PR:
- [ ] Internal ticket links to this PR
- [ ] Code diff has been done and been reviewed
- [ ] Tests are included and test edge cases
- [ ] Tests have been run locally and pass

**Reviewer 1:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

**Reviewer 2:**

Name:
- [ ] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [ ] The tests appropriately test the new code, including edge cases
- [ ] You have tried to break the code

### CODE REVIEW

**Comment by dczulada** [comment]
[2020JUN_QRDA_I_R1_STU5.2_Errata_List.xlsx](https://github.com/projecttacoma/cqm-reports/files/4860394/2020JUN_QRDA_I_R1_STU5.2_Errata_List.xlsx)

List of Errata changes are here

File: lib/qrda-export/catIII-r2-1/_header.mustache [hunk 1]
```diff
@@ -4,7 +4,7 @@
 <!-- US Realm Header Template Id -->
 <templateId root="2.16.840.1.113883.10.20.27.1.1" extension="2017-06-01"/>
 <!-- QRDA Category III Report - CMS (V4) -->
-<templateId root="2.16.840.1.113883.10.20.27.1.2" extension="2019-05-01"/>
+<templateId root="2.16.840.1.113883.10.20.27.1.2" extension="2020-05-01"/>
```
[the remark this answers is not in the document]
Comment (lmd59) [reply lib/qrda-export/catIII-r2-1/_header.mustache:7]: (Answer: yes)
